clar2wasm/tests/wasm-generation/control_flow.rs

```diff
@@ -24,16 +24,12 @@ fn begin_strategy() -> impl Strategy<Value = (String, PropValue, bool)> {
                 expressions.push(' ');
             }
 
-            expressions.push_str(&s);
+            expressions.push_str(&format!("{v}"));
```

Comment:
No need for a macro here, that's a `to_string`

Reply:
I thought macros are faster.

Reply:
No of course not! Macros add steps in the compilation process to create the code that will be placed here, and then this code still has to be executed in the end.